### DISCUSSION TITLE
{{date}} defauls should not include colon

### DISCUSSION THREAD

**Discussion by lduperval** [coddingtonbear/obsidian-web#122] [Ideas]
Hi,

The default date format includes colons for time. However, on Windows, colons are not allowed in file names so trying to save a new note with a date fails.

This could be fixed in handlebars.ts with:

let formatStr: string = "yyyy-MM-dd HHmmss";

or something similar.

**Comment by coddingtonbear** [comment]
BTW: You have full control over the date format used.

**Comment by coddingtonbear** [comment]
Closing since there is documentation in the wiki around how one might control the date format used.
